gymnasium/spaces/utils.py

```diff
@@ -329,7 +329,7 @@ def _unflatten_multidiscrete(
             f"{x} is not a concatenation of one-hot encoded vectors and can not be unflattened to space {space}. "
             "Not all valid samples in a flattened space can be unflattened."
         )
-    (indices,) = cast(type(offsets[:-1]), nonzero)
```

Comment:
Could you explain this change?

Reply:
Could we just go straight from np.nonzero to (indices,)?

Reply:
Makes sense, I've added it in the latest commit.